src/Nethermind/Nethermind.Taiko.Test/TxPoolContentListsTests.cs

```diff
@@ -101,6 +102,12 @@ public int[][] Test_TxLists_AreConstructed(
         return result.Data.Select(list => list.TxList.OfType<EIP1559TransactionForRpc>().Select(tx => (int)tx.Input![0]).ToArray()).ToArray();
     }
 
+    [Test]
+    public void ThisWillFail()
```

Comment:
This probably should be removed?

Reply:
Yes, I'm just testing some stuff since I think we're not running all tests. I'll revert these commits once I finish.